File: introduction/ITP1_2_B-Range.php
<?php
$input_lines = fgets(STDIN);

$abc = explode(' ', $input_lines);
$a = $abc[0];
$b = $abc[1];
$c = $abc[2];

if ($a < $b && $b < $c) {
  echo 'Yes';
} else {
  echo 'No';
}
?>

File: introduction/ITP1_2_B-Range.go
package main
import "fmt"
func main() {
  abc := make([]int, 3)
  for i:=0;i<3;i++ {
    fmt.Scanf("%d",&abc[i])
  }
  if abc[0] < abc[1] && abc[1] < abc[2] {
    fmt.Println("Yes")
  } else {
    fmt.Println("No")
  }
}
